src/pages/resources/Resources.js

```diff
@@ -1,24 +1,38 @@
-import ExternalLink from "../../components/externalLink/ExternalLink"
+import React from "react";
+import "./Resources.css";
+import { resources } from "../../constant";
 
-function Resources() {
+const showResources = () => {
+  return resources.map((resource, idx) => (
+    <li key={idx}>
+      <div className="image">
+        <img src={resource.logo} alt={resource.name} />
+      </div>
+      <div className="header">{resource.name}</div>
+      <div>{resource.description}</div>
+      <div className="link">
+        <i class="fa fa-link" aria-hidden="true">
+          :
+        </i>
+        <a href={resource.link} target="_blank" rel="noreferrer">
+          {resource.name}
+        </a>
+      </div>
+    </li>
+  ));
+};
+
+const Resources = () => {
   return (
-    <div>
-      <h1>Useful Resources</h1>
-      <div>The following links might be of interest to club members</div>
-      <br />
-      <div className="links">
-        <ExternalLink 
-          url='https://gdsc.community.dev/conestoga-college-waterloo-campus/' 
-          explanation='GDSC Portal - a place where all of our events are hosted' />
-        <ExternalLink 
-          url='https://conestogastudents.com/2021/09/01/coding-club/' 
-          explanation='CSI Club - register for the official GDSC Conestoga CSI club' />
-        <ExternalLink 
-          url='https://www.youtube.com/channel/UCBbpXl_fLG6dbu25nu2nRNA' 
-          explanation='YouTube - our channel with event recordings' />
+    <div className="resources_wrapper">
+      <div className="top">
+        <h1>OUR RESOURCES</h1>
+      </div>
+      <div className="bottom">
+        <ul className="resourcesList"> {showResources()}</ul>
```

Comment:
IMHO creating a separate constant configuration list and then mapping over it within a function is less readable than what we had previously: a hardcoded list of components with inline values. I would see value in your approach if we would have a 100 resources coming from some external API - in this case mapping over a dynamic list makes most sense. I'm not married to this idea though, so you are welcome to prove me wrong in the comments if you feel strongly about your aproach.

Reply:
I think its easier to edit our resources if its kept in a separate file, and just makes our html shorter

Reply:
ok, let's keep the function syntax, but in that case I'll ask you to do 2 things:
1. Clean up the `ExternalLink` component since it's not being used anywhere now
2. Inline the `map` on line 32 and make the function only return one resource (`showResource`). I think this way would be the best of 2 worlds